src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -1,23 +1,21 @@
 package edu.kis.powp.jobs2d.drivers;
 
 import edu.kis.powp.jobs2d.Job2dDriver;
-import edu.kis.powp.jobs2d.LoggerDriver;
 
 /**
  * Driver manager provides means to setup the driver. It also enables other
  * components and features of the application to react on configuration changes.
  */
 public class DriverManager {
 
-	private Job2dDriver currentDriver = new LoggerDriver();
+	private final Job2dDriver currentDriver = new ProxyDriver();
```

Comment:
You shouldn't force ProxyDriver directly. What if other feature require using their own "decorator". Should they remove yours? Revert DriverManager class.